### DISCUSSION TITLE
My feed filter settings for each feed individually

### DISCUSSION THREAD

**Discussion by ALMKM** [dailydotdev/daily#1217] [Feature Requests]
Would be great if we could not only choose tags for each additional feed we create but also block content for each feed and also disable enable categories.

Use case: You have a feed that u strictly only want for updates that are relevant to the tech stack you currently use in your project, with no distractions. Then you maybe also have another feed that you browse on lunch break where, for example, the meme category would be welcome. Or you are willing to explore or look at new libraries that you don't use atm, so you would maybe want to enable the comparison category there. 

And also for the block feature. You might have an article or a topic that accidentally got into one of your strictly filtered feeds. You definitely don't want to have it in, but you only want to block it for that specific feed, bc maybe in your more broad feed you would like to see that post.

**Comment by idoshamun** [comment]
Hi @ALMKM,

Thanks for the suggestion. We will consider it
